### PR TITLE
Creating an opensource lib

### PR DESCRIPTION
**Pull request by juliofalbo**
Go to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,239 @@
...
+For example, until [RabbitMQ 3.8](https://github.com/rabbitmq/rabbitmq-server/pull/1889), we do not have a property in the header to control the number of retries that Rabbit already did.
+
+RabbitMQ default behavior: 
+- If you didn't define the **[time-to-live](https://www.rabbitmq.com/ttl.html)** argument, RabbitMQ will try to requeue your message forever.
+- If you defined a ttl but didn't define a **[dlx](https://www.rabbitmq.com/dlx.html)**, after tll, RabbitMQ will remove your message to the queue and you will lost the message.
+- If you defined a ttl and a **[dlx](https://www.rabbitmq.com/dlx.html)**, after the ttl, RabbitMQ will send the message to the [**exchange**](https://medium.com/faun/different-types-of-rabbitmq-exchanges-9fefd740505d) defined in the dlx.
+ 
+![Default RabbitMQ Behavior](https://i.ibb.co/CnY7K4q/Screenshot-2019-05-15-at-10-37-44.png)
+ 
+> So, but if we want to have a growing tll (in case of instability, for example), and control the number of retries, how can we do this?
```
Comment (ehats) [inline README.md]: tll or ttl?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,239 @@
...
+
+public void sendMessage(final String message) {
+    rabbitTemplateHandler.getRabbitTemplate("some-event").convertAndSend(exchangeSomeEvent, routingKeySomeEvent, message);
+}
+```
+
+#### Listener
+
+To listeners is very simple too. The only thing that you need to do is annotate a method with the **RabbitListener** annotation and pass the name of **containerFactory**.
+> How I know the name of the correct container factory for each virtual host?
```
Comment (ehats) [inline README.md]: typo: How **do** I know

File: spring-rabbitmq-tuning-lib/pom.xml [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+        <version>2.1.4-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+
+    <artifactId>spring-rabbitmq-tuning-lib</artifactId>
+
+    <name>Spring RabbitMQ Tuning Library</name>
+    <description>This library was created to make it easier to configure RabbitMQ within Spring and to establish a pattern for queues architecture.</description>
+
+    <url>http://www.tradeshift.com</url>
```
Comment (ehats) [inline spring-rabbitmq-tuning-lib/pom.xml]: `https://tradeshift.com`

File: spring-rabbitmq-tuning-lib/pom.xml [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    <organization>
+        <name>Tradeshift</name>
+        <url>https://tradeshift.com</url>
+    </organization>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://www.opensource.org/licenses/mit-license.php</url>
```
Comment (ehats) [inline spring-rabbitmq-tuning-lib/pom.xml]: https

File: README.md (hunk 1 above)
Comment (juliofalbo) [reply README.md]: ttl, thank you!!!

File: README.md (hunk 2 above)
Comment (juliofalbo) [reply README.md]: Perfect!

**Pull request merged**